build-samples/build_samples.sh

```diff
@@ -25,13 +25,13 @@ fi
 
 make headers_install
 make \
-	CLANG=clang-${LLVM_VER} \
-	OPT=opt-${LLVM_VER} \
-	LLC=llc-${LLVM_VER} \
-	LLVM_DIS=llvm-dis-${LLVM_VER} \
-	LLVM_OBJCOPY=llvm-objcopy-${LLVM_VER} \
-	LLVM_READELF=llvm-readelf-${LLVM_VER} \
-	LLVM_STRIP=llvm-strip-${LLVM_VER} \
+	CLANG=clang-${LLVM_VERSION} \
```

Comment:
given that `LLVM` is already `-${LLVM_VERSION}`, we may just reaplce all the references below to something similar to:
` CLANG=clang${LLVM}`